OpenRA.Mods.Common/Util.cs

```diff
@@ -84,16 +84,17 @@ public static WAngle AngleDiffToStep(WAngle facing, int numFrames)
 			return new WAngle(a % step - step / 2);
 		}
 
-		public static WAngle GetInterpolatedFacing(WAngle facing, int facings, int interpolatedFacings)
+		/// <summary>Returns the angle that the closest facing sprite should be rotated by to achieve the closest interpolated facing.</summary>
```

Comment:
```suggestion
		/// <returns> The angle that the closest facing sprite should be rotated by to achieve the closest interpolated facing.</returns>
```

We should use this more often

Reply:
`summary` would by my preference here as it's the element you always have. I wouldn't usually employ `returns` unless there was something particularly interesting to say about it not already covered in the summary.